yarn.lock

```diff
@@ -9202,25 +9289,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"clipanion@npm:3.2.0":
-  version: 3.2.0
-  resolution: "clipanion@npm:3.2.0"
-  dependencies:
-    typanion: ^3.8.0
-  peerDependencies:
-    typanion: "*"
-  checksum: e28e6f0d48aecff86097823c604aa486082d76d2a5d3abc71069a0d9f3338af769fd7c6634b2f444c5b1aac0743b503325cc0b30552c094c01ebc602631b273d
-  languageName: node
-  linkType: hard
-
-"clipanion@npm:^4.0.0-rc.2":
-  version: 4.0.0-rc.3
-  resolution: "clipanion@npm:4.0.0-rc.3"
+"clipanion@npm:4.0.0-rc.4, clipanion@npm:^4.0.0-rc.2":
+  version: 4.0.0-rc.4
+  resolution: "clipanion@npm:4.0.0-rc.4"
```

Comment:
This is being required by the latest version of `verdaccio@5.x`, which is causing a TypeScript build failure.

Reply:
I had to set `skipLibCheck: true` in `@jupyterlab/buildutils` to suppress this failure. I can't think of any other workaround besides upgrading `verdaccio` to a non-deprecated major version, but that has to be done in a future PR.

Reply:
> I can't think of any other workaround

Instead of resolving to `verdaccio` 5.33 which depends on `clipanion` 4.0.0-rc.4, would depending on a slightly older version of ` verdaccio` solve the issue of `semver` and `tough-cookie` without introducing typescript compatibility issue (e.g. `verdaccio` 5.32 depends on clipanion` 4.0.0-rc.3 instead). Note that we previously had `4.0.0-rc.3` in the lockfile so it looks like that version is not affected.

Reply:
Thanks for that suggestion! I've made the updates & unset `skipLibCheck: true`.